src/topology/algebra/ring.lean

```diff
@@ -382,34 +382,34 @@ by { ext, rw h }
 instance : partial_order (ring_topology α) :=
 partial_order.lift ring_topology.to_topological_space $ ext
 
+@[simp] lemma to_topological_space_le {x y : ring_topology α} :
+  x.to_topological_space ≤ y.to_topological_space ↔ x ≤ y := iff.rfl
+
 local notation `cont` := @continuous _ _
 
-private def def_Inf (S : set (ring_topology α)) : ring_topology α :=
-let Inf_S' := Inf (to_topological_space '' S) in
-{ to_topological_space := Inf_S',
-  continuous_add       :=
-  begin
-    apply continuous_Inf_rng,
-    rintros _ ⟨⟨t, tr⟩, haS, rfl⟩, resetI,
-    have h := continuous_Inf_dom (set.mem_image_of_mem to_topological_space haS) continuous_id,
-    have h_continuous_id := @continuous.prod_map _ _ _ _ t t Inf_S' Inf_S' _ _ h h,
-    exact @continuous.comp _ _ _ (id _) (id _) t _ _ continuous_add h_continuous_id,
-  end,
-  continuous_mul       :=
-  begin
-    apply continuous_Inf_rng,
-    rintros _ ⟨⟨t, tr⟩, haS, rfl⟩, resetI,
-    have h := continuous_Inf_dom (set.mem_image_of_mem to_topological_space haS) continuous_id,
-    have h_continuous_id := @continuous.prod_map _ _ _ _ t t Inf_S' Inf_S' _ _ h h,
-    exact @continuous.comp _ _ _ (id _) (id _) t _ _ continuous_mul h_continuous_id,
-  end,
-  continuous_neg       :=
-  begin
-    apply continuous_Inf_rng,
-    rintros _ ⟨⟨t, tr⟩, haS, rfl⟩, resetI,
-    have h := continuous_Inf_dom (set.mem_image_of_mem to_topological_space haS) continuous_id,
-    exact @continuous.comp _ _ _ (id _) (id _) t _ _ continuous_neg h,
-  end }
+instance : has_Inf (ring_topology α) :=
```

Comment:
This does... not look like a very fun to use instance, especially given it's now not private.

Reply:
It's still the exact same definition for the infimum. All I did was cut the middleman private definition (which would have been a pain to unfold).

Reply:
Not to mention, most of this is just proofs, so it's not really a very inconvenient definition.